pkgs/development/libraries/opencv/3.x.nix

```diff
@@ -147,6 +149,6 @@ stdenv.mkDerivation rec {
     homepage = http://opencv.org/;
     license = stdenv.lib.licenses.bsd3;
     maintainers = with stdenv.lib.maintainers; [viric flosse mdaiter];
-    platforms = with stdenv.lib.platforms; linux;
+    platforms = with stdenv.lib.platforms; [ linux darwin ];
```

Comment:
Yeah, we also generally try to avoid passing in all of `darwin` and instead pass in the individual Frameworks we use. There's probably a better way to do this of course 😄